docs/wfforward.md

```diff
@@ -828,17 +825,25 @@ Let's take a look at how many roots are present in each marginal tree that came
 
 
 ```python
-ts = msprime.load_tables(nodes=n,edges=e,sites=s,mutations=m)
+ts = tc.tree_sequence()
 
 nroots=[]
 for i in ts.trees():
     nroots.append(i.num_roots)
+print(np.unique(nroots))
 p = sns.distplot(nroots, kde=False)
 p.set(xlabel="Number of roots",ylabel="Number");
 ```
 
+    [1 2]
+
+
+    /usr/lib/python3/dist-packages/matplotlib/axes/_axes.py:6462: UserWarning: The 'normed' kwarg is deprecated, and has been replaced by the 'density' kwarg.
```

Comment:
Do we want this in the output?

Reply:
No--any ideas how to suppress it?

Reply:
Upgrade seaborn, something like that?

Reply:
I just suppressed warnings for those plotting calls.